### PR TITLE
fix(web): headerの更新情報と操作を集約

### PR DESCRIPTION
**Pull request by hiroppy**
## Summary

- header の group 表示から最終更新日時を分離し、右側操作群の既存 crawler refresh ボタン左隣へ集約
- URL で選択された group の更新日時へ追従し、group 0件でも crawler refresh 操作を維持
- 削除済み workflow への link と更新ボタン重複を避け、Header の状態分岐・回帰テストを整理

## Linear Issue

- [HIR-139](https://linear.app/hiroppy/issue/HIR-139/header%E3%81%AE%E6%9B%B4%E6%96%B0%E6%99%82%E9%96%93%E8%A1%A8%E7%A4%BA%E3%81%A8%E6%9B%B4%E6%96%B0%E3%83%9C%E3%82%BF%E3%83%B3%E3%81%AE%E6%83%85%E5%A0%B1%E5%88%86%E6%95%A3%E3%82%92%E8%A6%8B%E7%9B%B4%E3%81%99)

### Selected Quality Characteristics

| Quality characteristic | Why it is relevant | Acceptance criteria |
| ---------------------- | ------------------ | ------------------- |
| 機能適合性（正確性） | 選択 group と更新日時・更新操作の対応が主変更 | URL 選択 group の日時を表示し、0件時も既存 crawler refresh が使える |
| Interaction capability（操作性・アクセシビリティ） | 関連情報の分散解消と header 操作性が目的 | 日時と更新ボタンが近接し、desktop/mobile で欠けず、更新ボタン重複がない |

### Test Techniques

| Test technique | Selection rationale | Expected coverage |
| -------------- | ------------------- | ----------------- |
| Decision table testing | group なし・単一・複数で表示が分岐する | group selector / 更新日時 / refresh 操作の主要組み合わせ |
| State transition testing | URL の group 変更で選択表示と更新日時が切り替わる | 有効 URL と未知 URL の default fallback |
| Boundary value analysis | 狭い mobile と desktop の代表境界で header 密度が変わる | 390×844 と 1440×900 のレイアウト |
| Checklist-based testing | accessible name、ボタン個数、既存操作を確認する | a11y と回帰の主要リスク |

### Primary Test Conditions

- group 0件でも既存 crawler refresh ボタンを表示する
- 複数 group では URL の group を選択し、その group 名と最終更新日時を表示する
- 未知の URL group では default group とその日時へ fallback する
- desktop/mobile で「更新日時 → crawler refresh ボタン」が右側操作群の先頭に収まる
- 削除済み `daily-update.yml` への link や更新ボタン重複がない

### Out-of-Scope Risks

- crawler refresh API 自体の状態遷移は既存テスト範囲であり、挙動変更なし
- 実データは対象外（匿名 demo DB のみ使用）

## Verification

- [x] Unit tests
- [x] Type checking
- [x] Lint
- [x] Storybook tests
- [x] E2E tests
- [x] Manual verification

### Commands Executed

```text
pnpm --filter @mf-dashboard/web test:unit — 33 files / 497 tests passed
pnpm --filter @mf-dashboard/web test:storybook — 78 files / 345 tests passed
pnpm turbo typecheck — 8 packages passed
pnpm lint — passed, warnings 0
pnpm format:check — passed
Playwright runtime walkthrough — HTTP 200, group transition passed, refresh button count 1, console errors 0
```

Manual verification media:

- desktop 1440×900: `.github/assets/hir-139-header-desktop.png`
- mobile 390×844: `.github/assets/hir-139-header-mobile.png`

### Checks Not Run

- N/A — all relevant local checks and runtime walkthrough were completed; latest CI is tracked below.

### CODE REVIEW

File: apps/web/src/components/layout/action-icons.tsx [hunk 1]
```diff
@@ -357,6 +365,55 @@ function formatStepMetadata(item: CrawlerRunStepDetails): string | null {
...
+    </div>
+  );
+}
+
+function getRefreshWorkflowUrl() {
+  const githubOrg = process.env.NEXT_PUBLIC_GITHUB_ORG;
+  const githubRepo = process.env.NEXT_PUBLIC_GITHUB_REPO;
+
+  return githubOrg && githubRepo
+    ? `https://github.com/${githubOrg}/${githubRepo}/actions/workflows/daily-update.yml`
```
Comment (hiroppy) [inline apps/web/src/components/layout/action-icons.tsx]: `daily-update.yml` へのリンクを復活させていますが、このリポジトリには `.github/workflows/ci.yml` しかなく、過去に `daily-update.yml` は削除されています。`NEXT_PUBLIC_GITHUB_ORG/REPO` が設定された環境ではヘッダーに存在しない Actions workflow へのリンクが表示されるので、既存の更新導線に向けるか、このリンク生成を削除してください。

File: apps/web/src/components/layout/header.test.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import { afterEach, describe, expect, it, vi } from "vitest";
+import { Header } from "./header";
+import { SidebarProvider } from "./sidebar-context";
+
+const workflowUrl = "https://github.com/example/example/actions/workflows/daily-update.yml";
+
+vi.mock("next/navigation", () => ({
```
Comment (hiroppy) [inline apps/web/src/components/layout/header.test.tsx:25]: この mock は `usePathname` だけを返しますが、`Header` は `ActionIcons` → `RefreshControl` 経由で `useRouter()` も呼ぶため、追加されたテストが `No "useRouter" export is defined on the "next/navigation" mock` で失敗します。`useRouter: () => ({ refresh: vi.fn() })` なども mock してください。

**Comment by hiroppy** [comment]
モバイル版も更新ボタンは一番左

File: apps/web/src/components/layout/header.test.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    render(
+      <SidebarProvider>
+        <Header groups={[]} defaultGroupId={null} refreshWorkflowUrl={workflowUrl} />
+      </SidebarProvider>,
+    );
+
+    expect(
+      screen.getByRole("link", { name: "データ更新ワークフローを開く" }).getAttribute("href"),
+    ).toBe(workflowUrl);
+  });
```
[the remark this answers is not in the document]
Comment (hiroppy) [reply apps/web/src/components/layout/header.test.tsx:53]: 対応しました。URL の有効 group と未知 group の default fallback を追加し、それぞれ対応する更新日時と既存 crawler refresh ボタンを検証しています。削除済み workflow link は別 feedback に従って撤去したため、URL assertion は有効な crawler refresh 操作の存在確認へ置き換えました。unit 33 files / 497 tests passed です。

File: apps/web/src/components/layout/action-icons.tsx (hunk 1 above)
Comment (hiroppy) [reply apps/web/src/components/layout/action-icons.tsx]: 対応しました。`daily-update.yml` の link 生成と追加 refresh icon を削除し、最終更新日時を既存 `RefreshControl` の左隣へ配置しました。更新操作は現行 crawler refresh API のみとなり、runtime で button 1個を確認しています。

File: apps/web/src/components/layout/header.test.tsx (hunk 2 above)
Comment (hiroppy) [reply apps/web/src/components/layout/header.test.tsx:25]: 対応しました。`useRouter` と `EventSource` を mock し、実際に初回失敗を再現後、unit 33 files / 497 tests passed を確認しました。

File: apps/web/src/components/layout/action-icons.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (hiroppy) [reply apps/web/src/components/layout/action-icons.tsx]: 対応しました。存在しない workflow shortcut は削除し、日時ラベルを現行 crawler refresh button と一体化しました。Docker/環境変数への追加配線は不要です。

File: apps/web/src/components/layout/header.test.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import { afterEach, describe, expect, it, vi } from "vitest";
+import { Header } from "./header";
+import { SidebarProvider } from "./sidebar-context";
+
+const workflowUrl = "https://github.com/example/example/actions/workflows/daily-update.yml";
+
+vi.mock("next/navigation", () => ({
+  usePathname: () => "/",
+}));
```
[the remark this answers is not in the document]
Comment (hiroppy) [reply apps/web/src/components/layout/header.test.tsx:28]: 対応しました。`useRouter().refresh` と `EventSource` を test で stub し、Header の全 unit test を含む 497 tests が通過しています。

**Comment by hiroppy** [comment]
Commit: https://github.com/hiroppy/mf-dashboard/commit/9bd73019eea9c4603023d79472900a24324bd547

### Desktop: 更新日時と更新ボタンが右側操作群の先頭にまとまって表示される

header-desktop

### Mobile: 狭い幅でも更新日時・更新ボタン・通知・Money Forward 導線が欠けずに表示される

header-mobile
